mesa/mesaPROTON_OC.py

```diff
@@ -587,6 +587,19 @@ def df_to_dict(self, df):
                 dic[key] = subdic
         return dic
 
+    def setup_education_levels(self):
+        """
+        Modify the self.education_levels attribute in-place. Given 4 levels of education,
+        for each level returns the correct amount of schools, based on the number of agents.
+        """
+        self.list_schools = self.read_csv_city("schools").values.tolist()
+        for education_level in self.list_schools:
+            education_level[3] = np.ceil((education_level[3]/education_level[4])*self.initial_agents)
+            education_level.remove(education_level[4])
```

Comment:
Good until here. What happened to the last part?
```
   set education-levels lput (list index new-row) education-levels
    set index index + 1
  ]
  set education-levels table:from-list education-levels
```
The index has disappeared, and there's no table (dictionary) structure.

Reply:
I was undecided which structure to use. I thought a list of lists might be useful. But it's better to stay consistent and use a dictionary.